See: https://github.com/golang/go/issues/17276

commit c5434f2973a87acff76bac359236e690d632ce95
Author: Alberto Donizetti <alb.donizetti@gmail.com>
Date:   Thu Sep 29 13:59:10 2016 +0200

    time: update test for tzdata-2016g
    
    Fixes #17276
    
    Change-Id: I0188cf9bc5fdb48c71ad929cc54206d03e0b96e4
    Reviewed-on: https://go-review.googlesource.com/29995
    Reviewed-by: Brad Fitzpatrick <bradfitz@golang.org>
    Run-TryBot: Brad Fitzpatrick <bradfitz@golang.org>
    TryBot-Result: Gobot Gobot <gobot@golang.org>

diff --git a/src/time/time_test.go b/src/time/time_test.go
index 68236fd64d..2e47d0819d 100644
--- a/src/time/time_test.go
+++ b/src/time/time_test.go
@@ -925,8 +925,11 @@ func TestLoadFixed(t *testing.T) {
 	// but Go and most other systems use "east is positive".
 	// So GMT+1 corresponds to -3600 in the Go zone, not +3600.
 	name, offset := Now().In(loc).Zone()
-	if name != "GMT+1" || offset != -1*60*60 {
-		t.Errorf("Now().In(loc).Zone() = %q, %d, want %q, %d", name, offset, "GMT+1", -1*60*60)
+	// The zone abbreviation is "-01" since tzdata-2016g, and "GMT+1"
+	// on earlier versions; we accept both. (Issue #17276).
+	if !(name == "GMT+1" || name == "-01") || offset != -1*60*60 {
+		t.Errorf("Now().In(loc).Zone() = %q, %d, want %q or %q, %d",
+			name, offset, "GMT+1", "-01", -1*60*60)
 	}
 }
 
